app/services/migration/migrators/contract.rb

```diff
@@ -42,12 +42,9 @@ def call
           contract_template = ::ContractTemplate.find_or_initialize_by(ecf_id: ecf_contract.id)
           contract_template.update!(ecf_contract.attributes.slice(*SHARED_ATTRIBUTES))
 
-          contract = ::Contract.find_or_initialize_by(
-            statement_id:,
-            course_id:,
-          )
+          contract = ::Contract.find_or_initialize_by(ecf_id: ecf_contract.id)
```

Comment:
not sure we need this, we are using the `ecf_id` in the contract template which maps to the NPQ contract id in ecf, the join table i.e. contract table shouldn't have an `ecf_id` 🤔 as it doesn't map back, I think we can remove/ignore it?

Reply:
yeah that sounds sensible I'll remove it 👍